src/site/stages/build/drupal/graphql/navigation-fragments/sidebar.nav.graphql.js

```diff
@@ -50,11 +33,39 @@ module.exports = `
                 url {
                   path
                 }
+                links {
+                  label
+                  expanded
+                  description
+                  url {
+                    path
+                  }
+                  links {
+                    label
+                    expanded
+                    description
+                    url {
+                      path
+                    }
+                  }
+                }
               }
             }
           }
-        }
-      }
     }
-  }
+`;
+
+function queryFilter(menuName) {
+  return `
+      menuByName(name: "${menuName}")
+    `;
+}
+
+module.exports = `
+    healthcareHubSidebarQuery: ${queryFilter('health-care-benefits-hub')} {
```

Comment:
Is it possible to pull a list of all the menus instead of having to add queries for each one?

Reply:
no unfortunately, can only get them by name for now.